jenkins/bisect.jpl

```diff
@@ -390,16 +390,32 @@ def findMergeBase(kdir, good, bad) {
             script: "git merge-base --is-ancestor ${base} HEAD")
 
         if (good_base != 0) {
-            def ref = "${params.REF_KERNEL_TREE}/${params.REF_KERNEL_BRANCH}"
+
+            dir(kci_core) {
+                if (!(ref_kernel_url && ref_kernel_tree && params.ref_kernel_branch)) {
```

Comment:
`params.ref_kernel_branch` is wrong - in fact these should be `ref_url`, `ref_tree` and `ref_branch`.

Reply:
Pushed missing corrections